sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 7.0.0b3 (Unreleased)
+## 7.0.0b3 (2020-06-08)
 
 **New Features**
 
+* Added support for management of queue entities.
+    - Use `azure.servicebus.management.ServiceBusManagementClient` (`azure.servicebus.management.aio.ServiceBusManagementClient` for aio) to perform queue-related CRUD operations.
```

Comment:
Is it more explicit to say "... to create, update, delete, and list queues of a ServiceBus namespace"?

Reply:
updated!